api/controllers/AuthController.js

```diff
@@ -60,10 +60,48 @@ module.exports = {
           res.view(403, {error: err});
           return;
         }
-        if (url) {
-          return res.redirect(url);
-        }
-        return res.redirect('/');
+
+        Reddit.checkModeratorStatus(
```

Comment:
We could have a page only accessible to mods (e.g. hq.porygon.co/sessionclear/<username>) that clears any user's session.